### PR TITLE
Remove any non alphabetical character from avatar initials

### LINKED ISSUE #2674: Avatar initials should only use letters
I added a second blog feed for myself to test the work that @dbelokon is doing with YouTube feeds, and the feed's name is "David Humphrey (YouTube)".  Our avatar logic is using `(` as my final initial:

Screen Shot 2022-01-19 at 10 14 07 AM

We should probably have this only use letters.  I'm not sure what the right logic would be here to have it get "DH" vs. "DY" for YouTube.

### PR DESCRIPTION
**Pull request by joelazwar**
## Issue This PR Addresses

Fixes #2674 

image

image

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/web/src/components/Posts/PostAvatar.tsx
Comment (humphd) [inline src/web/src/components/Posts/PostAvatar.tsx:51]: Did you test this?  It still does the same thing with `David Humphrey (YouTube)`.  You need to add `/g` to this regex to deal with all matches globally.

**Review by humphd — CHANGES_REQUESTED** [review]

File: src/web/src/components/Posts/PostAvatar.tsx
Comment (tpmai22) [reply src/web/src/components/Posts/PostAvatar.tsx:51]: @humphd can you explain on how to test it ? cause I trying to test the your Youtube.

Comment (joelazwar) [reply src/web/src/components/Posts/PostAvatar.tsx:51]: my fault, im having a little trouble getting the video feeds showing up. will update it with test screenshots

**Review by humphd — DISMISSED** [review]

**Review by DukeManh — APPROVED** [review]

**Review by tpmai22 — DISMISSED** [review]

**Review by cindyledev — DISMISSED** [review]

**Comment by joelazwar** [comment]
sorry I was having trouble rebasing and ended up force pushing, could you guys approve it again?

**Review by cindyledev — DISMISSED** [review]

**Review by tpmai22 — DISMISSED** [review]

**Comment by humphd** [comment]
Let's rebase this properly before we merge, we shouldn't have `master` merged *into* your branch:

Screen Shot 2022-01-28 at 3 54 10 PM

See https://github.com/Seneca-CDOT/telescope/blob/master/docs/git-workflow.md#squashing-commits, and get the sheriffs to help if you're stuck (cc @DukeManh, @AmasiaNalbandian).

**Review by cindyledev — APPROVED** [review]

**Review by humphd — APPROVED** [review]
Looks good.

Please squash and rebase this:

```sh
git checkout master
git pull upstream master
git checkout issue-2674
git rebase master -i
...edit your commits, use squash for the second, save and exit
git push origin issue-2674 -f
```

**Comment by joelazwar** [comment]
sorry, I just woke up and saw that you approved the changes, but my page hadn't loaded your message and I thought I could just press the squash and merge button. Honest mistake, can I roll it back somehow?

**Comment by humphd** [comment]
OK.  Let's do it properly next time.

**Pull request merged**
